arch/arm64/boot/dts/qcom/qcm6490-shift-otter.dts

```diff
@@ -523,7 +523,24 @@
 &i2c13 {
 	status = "okay";
 
-	/* focaltech FT3658U @ 38 */
+	touchscreen@38 {
+		compatible = "focaltech,fts";
+		reg = <0x38>;
+
+		/* FIXME!!!! */
+		interrupt-parent = <&tlmm>;
+		interrupts = <81 0x2008>;
```

Comment:
I guess with FIXME you mean the 0x2008? Please specify ;) Also since it's not going upstream like this anyways imo if it works, remove the fixme